src/c++/perf_analyzer/genai-perf/docs/tutorial.md

```diff
@@ -26,9 +26,15 @@ OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
Small nit (no need to change anything, just wanted to check that you saw this):

https://github.com/triton-inference-server/client/commit/8f3dbc3546acd69e9887080ae34960c379421321#r143056360

Reply:
Yeah I'm actually planning on using these anchors for the automated testing. Plus I think the heading is going to be tweaked again (there is more cleanup I want to do) and I didn't want to have to keep updating the anchor.